<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="redSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>67</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="greenSpinBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>67</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="blueSpinBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>67</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
